charts/shoot-core/components/charts/node-problem-detector/values.yaml

```diff
@@ -86,6 +86,10 @@ affinity: {}
 nodeSelector: {}
 
 env:
+  - name: NODE_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: spec.nodeName
```

Comment:
We already have the env var here:
https://github.com/gardener/gardener/blob/2e993388559b10582f1fc00623cb371b5262df65/charts/shoot-core/components/charts/node-problem-detector/templates/daemonset.yaml#L49-L53
Why is this added a second time in the values?

This way, the env var will be added twice 😄 
```yaml
        env:
        - name: NODE_NAME
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: spec.nodeName
        - name: NODE_NAME
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: spec.nodeName
```